packages/ui/src/common/api/queries/__generated__/baseTypes.generated.ts

```diff
@@ -38,24 +38,6 @@ export type ApplicationFormQuestion = BaseGraphQlObject & {
   applicationformquestionanswerquestion?: Maybe<Array<ApplicationFormQuestionAnswer>>
 }
 
-export type ApplicationFormQuestionAnswer = BaseGraphQlObject & {
-  __typename: 'ApplicationFormQuestionAnswer'
-  id: Scalars['ID']
-  createdAt: Scalars['DateTime']
```

Comment:
I might be missing something in the GH UI. The generated file should be an outcome of the `schema.graphql` or a `../queries/*.graphql` files. However, If you didn't change the `schema.graphql` file it is OK as you've generated the other queries.

Just to double-check :)